assets/monitoring/prometheus/v1/recording.prometheusrule.yaml

```diff
@@ -1,229 +1,289 @@
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
-  name:  "{{ .scyllaDBMonitoringName }}-recoding"
+  name: "{{ .scyllaDBMonitoringName }}-recoding"
```

Comment:
typo - file name says recording, this here says recoding

Reply:
historical but fixed